src/robot/parsing/model/statements.py

```diff
@@ -944,14 +944,16 @@ def patterns(self):
     def variable(self):
         return self.get_value(Token.VARIABLE)
 
-    def validate(self):
+    def validate(self):    
```

Comment:
This kind of whitespace is a bit annoying because it adds unnecessary changes to diff now and also in the future when it will be removed. I'll remove it myself now, but please check these a bit more carefully in the future. Easiest is configuring editor to automatically remove trailing spaces (and to add a newline at the end of a file). #nitpicking